libsdhcdrivers/src/sdhc.c

```diff
@@ -785,7 +775,7 @@ int sdhc_init(void *iobase, const int *irq_table, int nirqs, ps_io_ops_t *io_ops
     sdhc->cmd_list_head = NULL;
     sdhc->cmd_list_tail = &sdhc->cmd_list_head;
     sdhc->version = ((readl(sdhc->base + HOST_VERSION) >> 16) & 0xff) + 1;
-    printf("SDHC version %d.00\n", sdhc->version);
+    ZF_LOGI("SDHC version %d.00", sdhc->version);
```

Comment:
make this `ZF_LOGD()` also becuase the host controller's support is a very static thing in the end? In `mmc_decode_csd()` we use `ZF_LOGD()` also for the card settings also. Should we use `ZF_LOGI()` there then also, so usage of incompatible cards can be seen form the logs?

Reply:
Switched ZF_LOGI() to ZF_LOGD() with the latest commits.